src/app/components/algorithms/algorithm-view/algorithm-view.component.scss

```diff
@@ -0,0 +1,14 @@
+.main-container {
+  margin: 16px;
```

Comment:
Boostrap margins might be better [see here](https://getbootstrap.com/docs/4.0/utilities/spacing/)

Reply:
Done